### PR TITLE
certbot, acme: 0.24.0 -> 0.30.2

### PR DESCRIPTION
**Pull request by dtzWill**
(same source)

###### Motivation for this change

Have not reviewed changelogs or tested on a server.

Should at least make sure the 'acme' test passes :).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dtzWill** [comment]
eep actually saw a build failure when trying it again, investigating...

File: pkgs/tools/admin/certbot/default.nix [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 pythonPackages.buildPythonApplication rec {
```
Comment (dotlambda) [inline pkgs/tools/admin/certbot/default.nix]: While we're at it, can we use `python3Packages`?

**Comment by dotlambda** [comment]
@GrahamcOfBorg build nixosTests.acme

**Review by dotlambda — APPROVED** [review]
Did not test it on my server, though.

**Pull request merged**